tests/server-integration/Utils.hs

```diff
@@ -102,8 +102,8 @@ scrubDates =
 cofferTest :: IO () -> IO ()
 cofferTest test = deleteRecords >> test
 
-setField :: Text -> Text -> Text -> IO (JsonResponse Value)
-setField path name contents =
+setField :: Text -> Text -> Maybe Bool -> Text -> IO (JsonResponse Value)
```

Comment:
Why not straight `Maybe Visibility`

Reply:
Because in #105 it says:
> And let's not use coffer's core data types/encoders/decoders, let's use json literals (e.g. with [aesonQQ](https://hackage.haskell.org/package/aeson-2.0.3.0/docs/Data-Aeson-QQ-Simple.html#v:aesonQQ))

We don't import anything from coffer core so just for consistency I don't import `Visibility` too

Reply:
I think I will change `Bool` to `Text` to unify the style with function `changeFieldVisibility`